.github/workflows/workflow.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install flake8==3.8.1 flake8-bugbear flake8-comprehensions isort==4.3.21
+          python -m pip install flake8==3.8.1 flake8-bugbear flake8-comprehensions isort==4.3.21 onnxruntime
```

Comment:
this test is only about linter and does not require this dependency

Reply:
It seems to move the import around depending on whether or not the package is installed. If the package is not installed it's moved below the detectron2 imports. If it is installed then it's moved above the detectron2 imports.